app/models/manageiq/providers/hawkular/datawarehouse_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,105 @@
+class ManageIQ::Providers::Hawkular::DatawarehouseManager::EventCatcher::Runner <
```

Comment:
This work for us (`hawkular/middleware`). `app/models/manageiq/providers/hawkular/common/` could be the right place. `app/controllers/mixins/` in the UI repo looks to be more suitable for controller stuff. Perhaps the `/app/models/mixins/` can work as well.